content/en/docs/ops/integrations/prometheus/index.md

```diff
@@ -146,7 +167,18 @@ tls_config:
   ca_file: /etc/prom-certs/root-cert.pem
   cert_file: /etc/prom-certs/cert-chain.pem
   key_file: /etc/prom-certs/key.pem
-  insecure_skip_verify: true  # Prometheus does not support Istio security naming, thus skip verifying target pod ceritifcate
+  insecure_skip_verify: true  # Prometheus does not support Istio security naming, thus skip verifying target pod certificate
+{{< /text >}}
+
+If you are using ServiceMonitors and PodMonitors, adjust the monitors from above as follows:
```

Comment:
These instructions incorrectly assume that a `PodMonitor` may have a file reference (as opposed to a secret ref), but Prometheus Operator doesn't allow for that, unlike with `ServiceMonitor`.

https://github.com/prometheus-operator/prometheus-operator/issues/3922